src/common/HeartbeatMap.h

```diff
@@ -79,12 +80,12 @@ class HeartbeatMap {
  private:
   CephContext *m_cct;
   RWLock m_rwlock;
-  time_t m_inject_unhealthy_until;
+  unsigned m_inject_unhealthy_until;
```

Comment:
if this is a time_point instead wouldn't that avoid most of the casts?

Reply:
agree, would be easier if this is a `time_point`.